README.md

```diff
@@ -571,6 +571,20 @@ JSON data
 
 You specify `'json'` encoding for both keys and/or values, you can then supply JavaScript objects to LevelUP and receive them from all fetch operations, including ReadStreams. LevelUP will automatically *stringify* your objects and store them as *utf8* and parse the strings back into objects before passing them back to you.
 
+If you are encoding buffers and would like a solution which automatically decodes the buffers back to their original form, you can use [`json-buffer`](https://github.com/dominictarr/json-buffer) as a custom encoding like this:
```

Comment:
Can you clarify "encoding buffers", to avoid confusing newcomers? This is about encoding buffers "as Base64 JSON strings", or something like that. It's an important distinction from the existing `binary` encoding.

Perhaps this belongs on the wiki though?